index.js

```diff
@@ -164,7 +164,12 @@ function fastifyMultipart (fastify, options = {}, done) {
 
   fastify.addContentTypeParser('multipart', setMultipart)
   fastify.decorateRequest(kMultipartHandler, handleMultipart)
+
+  fastify.decorateRequest('parts', getMultipartIterator)
+  // keeping multipartIterator to avoid bumping a major
```

Comment:
wonder if this could be done by using some kind of getter that would be emitting warnings when used. although that might require support on fastify level

Reply:
maybe just a wrapping function would suffice here, actually

Reply:
This has been released for so little that I don't think we should bother with a full blown deprecation.

Reply:
Still a new major.